src/validations/mta/mtaValidations.ts

```diff
@@ -24,44 +19,34 @@ export function watchMtaYamlAndDevExtFiles(disposables: Disposable[]): void {
     `**/{${MTA_YAML},${DEV_MTA_EXT}}`
   );
 
-  mtaFileWatcher.onDidChange(
-    (uri) => updateMtaDiagnostics(uri, disposables),
-    undefined,
-    disposables
-  );
+  const listener: (e: Uri) => Promise<void> = (uri: Uri) =>
+    updateMtaDiagnostics(uri, disposables);
+  const diagnosticParams = [undefined, disposables];
```

Comment:
I don't like this... these are 2 separate parameters, why are you putting them in an array and then deconstructing it everywhere?